index.html

```diff
@@ -77,6 +77,230 @@ <h3>LET'S CONNECT</h3>
         </li>      
        </ul>
     </section>
+    <section class="portfolio" id="portfolio">
+      <nav class="portfolio-border">
+        <img
+          class="portfolio-image"
+          alt="portfolio project"
+          src="Snapshoot_Portfolio1.png"
+        />
+        <div class="project-section">
+          <h3 class="project-name">Tonic</h3>
+          <nav class="project-info">
+            <h4 class="project-client">CANOPY</h4>
+            <img src="Counter.png" class="counter" alt="counter" />
+            <span class="programming-language">Back End Dev</span>
+            <img src="Counter.png" class="counter" alt="counter" />
+            <span class="year">2015</span>
+          </nav>
```

Comment:
- [ ] [OPTIONAL] Please consider replacing this `nav` tag with a more natural HTML tag at this position for semantic reasons. The semantics of the `nav` element is that of providing links, but this block does not contain links. Read more about Nav tag [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)